### ISSUE TITLE
Can not run with `apollo-server@3`

### ISSUE THREAD

**Issue by heeju** [zapier/apollo-server-integration-testing#19]
I upgraded `apollo-server-express` to 3.0.2
then,  I ran test with `apollo-server-integration-testing` and got the error.

```
error TS2740: Type 'ApolloServer<ExpressContext>' is missing the following properties from type 'ApolloServer': supportsSubscriptions, supportsUploads, setGraphQLPath, willStart, and 2 more.

    14       apolloServer
             ~~~~~~~~~~~~

      node_modules/apollo-server-integration-testing/dist/index.d.ts:9:5
        9     apolloServer: ApolloServer;
              ~~~~~~~~~~~~
        The expected type comes from property 'apolloServer' which is declared here on type 'TestClientConfig'
```

**Comment by thomas88** [comment]
Can you try my pull request https://github.com/zapier/apollo-server-integration-testing/pull/20 and provide feedback?

**Comment by 1garo** [comment]
Being having the same problem, @thomas88 if you guide me on how to point the version of the lib to yours I can test it, it has being a block for me for a week know

**Comment by thomas88** [comment]
@1garo As end user I think the most simple solution is to add the following in `package.json`:
```json
  "resolutions": {
    "apollo-server-express": "3.1.2",
  }
```

**Comment by 1garo** [comment]
I continue with the following type error:

![image](https://user-images.githubusercontent.com/44412643/141491143-62da6d0f-cad6-4a27-b2d0-5a729392aea1.png)

**Comment by adamwdennis** [comment]
I'm also getting the above errors with `apollo-server-express@3.4.0`

**Comment by guilherme90** [comment]
Hello... Do we have any solution...? I'm getting the same errors too.

**Comment by Sami-5** [comment]
Yeah I'm getting this too, any idea when the fix will come as it's blocking us from upgrading?

**Comment by yasharma** [comment]
i was facing the same issue and fixed by following this #20 https://github.com/zapier/apollo-server-integration-testing/pull/20#issuecomment-1004245811
